docs/eventing/02-01-eventing-architecture.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Architecture
+---
+
+Kyma Eventing uses NATS to implement an Event Publisher Proxy and Eventing Controller which work together to process and deliver events in Kyma.
+
+## Event Publisher Proxy
+
+The Event Publisher Proxy component receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless Functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway. It also fetches a list of subscriptions for a connected application.
```

Comment:
The Event Publisher Proxy component receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless Functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway. It also fetches a list of subscriptions for a connected application.
```suggestion
The Event Publisher Proxy component receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless Functions) and redirects them to the NATS server. It also fetches a list of subscriptions for a connected application.
```